### ISSUE TITLE
Not installing, npm error

### ISSUE THREAD

**Issue by marcemira** [plusacht/ember-bootstrap-datetimepicker#51]
Tried `ember install ember-bootstrap-datetimepicker` and I get:

```
shasum check failed for /tmp/npm-27982-cf052115/registry.npmjs.org/ember-bootstrap-datetimepicker/-/ember-bootstrap-datetimepicker-0.6.0.tgz
Expected: 7bbe4cbf7b0e843d2d331a78af2075f38a6f1c0b
Actual:   d113a802b1007efd5a0d691b1f53a6c84fdab59d
From:     https://registry.npmjs.org/ember-bootstrap-datetimepicker/-/ember-bootstrap-datetimepicker-0.6.0.tgz
Error: shasum check failed for /tmp/npm-27982-cf052115/registry.npmjs.org/ember-bootstrap-datetimepicker/-/ember-bootstrap-datetimepicker-0.6.0.tgz
Expected: 7bbe4cbf7b0e843d2d331a78af2075f38a6f1c0b
Actual:   d113a802b1007efd5a0d691b1f53a6c84fdab59d
From:     https://registry.npmjs.org/ember-bootstrap-datetimepicker/-/ember-bootstrap-datetimepicker-0.6.0.tgz
    at /var/www/diezpe/diezpe-dashboard/node_modules/ember-cli/node_modules/npm/node_modules/sha/index.js:38:8

```

**Comment by jasonmit** [comment]
I can reproduce this.  @claudiocro can you republish

**Comment by claudiocro** [comment]
I released 1.0.1.
